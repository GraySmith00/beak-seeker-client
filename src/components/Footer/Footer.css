.footer {
  display: flex;
  justify-content: space-around;
  align-items: center;
  position: fixed;
  bottom: 0;
  width: 375px;
  background: #1b2026; }
  .footer .nav-link {
    display: flex;
    justify-content: center;
    align-items: center;
    width: 100%;
    border-right: 1px solid black;
    font-size: 1.5rem;
    text-decoration: none;
    color: #fff; }
