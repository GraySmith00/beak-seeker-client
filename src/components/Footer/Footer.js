import React from 'react';
import { NavLink } from 'react-router-dom';

const Footer = props => {
  return (
    <div className="footer">
      <NavLink exact to={'/home'}>
        <p>
          <i className="fas fa-home" />
        </p>
      </NavLink>
    </div>
  );
};

export default Footer;
