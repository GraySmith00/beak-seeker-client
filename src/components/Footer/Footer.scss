.footer {
  display: flex;
  justify-content: space-around;
  align-items: center;

  position: fixed;
  bottom: 0;
  width: 375px;

  background: green;

  .nav-link {
    display: flex;
    justify-content: center;
    align-items: center;

    width: 100%;
    border-right: 1px solid black;
  }
}
