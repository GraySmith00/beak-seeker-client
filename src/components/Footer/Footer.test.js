import React from 'react';
import { shallow } from 'enzyme';

import Footer from './Footer';

describe('Footer component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it('it should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
