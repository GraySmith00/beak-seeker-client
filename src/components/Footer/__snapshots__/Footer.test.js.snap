// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component it should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          aria-current="page"
          className="nav-link"
          exact={true}
          to="/home"
>
          <p>
                    <i
                              className="fas fa-home"
                    />
          </p>
</NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          className="nav-link"
          exact={true}
          to="/hotspots"
>
          <p>
                    <i
                              className="fas fa-map-marked-alt"
                    />
          </p>
</NavLink>,
        <NavLink
          activeClassName="active"
          aria-current="page"
          className="nav-link"
          exact={true}
          to="/logout"
>
          <p>
                    <i
                              className="fas fa-sign-out-alt"
                    />
          </p>
</NavLink>,
      ],
      "className": "footer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <p>
            <i
                        className="fas fa-home"
            />
</p>,
          "className": "nav-link",
          "exact": true,
          "to": "/home",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <i
              className="fas fa-home"
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fas fa-home",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          "type": "p",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <p>
            <i
                        className="fas fa-map-marked-alt"
            />
</p>,
          "className": "nav-link",
          "exact": true,
          "to": "/hotspots",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <i
              className="fas fa-map-marked-alt"
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fas fa-map-marked-alt",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          "type": "p",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <p>
            <i
                        className="fas fa-sign-out-alt"
            />
</p>,
          "className": "nav-link",
          "exact": true,
          "to": "/logout",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <i
              className="fas fa-sign-out-alt"
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fas fa-sign-out-alt",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          "type": "p",
        },
        "type": [Function],
      },
    ],
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="nav-link"
            exact={true}
            to="/home"
>
            <p>
                        <i
                                    className="fas fa-home"
                        />
            </p>
</NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="nav-link"
            exact={true}
            to="/hotspots"
>
            <p>
                        <i
                                    className="fas fa-map-marked-alt"
                        />
            </p>
</NavLink>,
          <NavLink
            activeClassName="active"
            aria-current="page"
            className="nav-link"
            exact={true}
            to="/logout"
>
            <p>
                        <i
                                    className="fas fa-sign-out-alt"
                        />
            </p>
</NavLink>,
        ],
        "className": "footer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": <p>
              <i
                            className="fas fa-home"
              />
</p>,
            "className": "nav-link",
            "exact": true,
            "to": "/home",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fas fa-home"
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fas fa-home",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "p",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": <p>
              <i
                            className="fas fa-map-marked-alt"
              />
</p>,
            "className": "nav-link",
            "exact": true,
            "to": "/hotspots",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fas fa-map-marked-alt"
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fas fa-map-marked-alt",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "p",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": <p>
              <i
                            className="fas fa-sign-out-alt"
              />
</p>,
            "className": "nav-link",
            "exact": true,
            "to": "/logout",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fas fa-sign-out-alt"
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fas fa-sign-out-alt",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "p",
          },
          "type": [Function],
        },
      ],
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
